### PR TITLE
Refactor testcase for config action

### PR DESCRIPTION
**Pull request by ganeshrn**
*  Add testcase to test config action in interactive
   and stdout mode
*  Move integration test into it's own folder
*  Add fixtures folder

### CODE REVIEW

**Comment by cidrblock** [comment]
recheck

File: tools/test-setup.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -xe
```
Comment (pabelanger) [inline tools/test-setup.sh:1]: This isn't needed any more we've landed changes to zuul, to always use the latest version of podman.

Comment (ganeshrn) [reply tools/test-setup.sh:1]: Tracked with issue https://github.com/ansible/ansible-navigator/issues/126

**Comment by ganeshrn** [comment]
Slower integration test run issue tracked via https://github.com/ansible/ansible-navigator/issues/127

**Pull request merged**